### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by ssolmer**
- Removed .NET Core specific syntax
  - Added BitBucket Repo owner
  - Added Thread Sleeps to prevent Github blocking per gregsdennis
  - Keep console window open after complete

### CODE REVIEW

File: BitToGithub/Program.cs [hunk 1]
```diff
@@ -23,7 +25,9 @@ static void Main(string[] args)
             var isAllDefined = new[] { bbUsername, bbPassword, bbRepoName, githubUsername, githubPassword, githubRepoName, githubRepoOwnerName }.All(param => string.IsNullOrEmpty(param) == false);
             if (isAllDefined)
             {
-                string bbRepoIssuesUrl = "https://api.bitbucket.org/1.0/repositories/\{bbUsername}/\{bbRepoName}/issues";
+                //string bbRepoIssuesUrl = "https://api.bitbucket.org/1.0/repositories/\{bbUsername}/\{bbRepoName}/issues";
```
Comment (tugberkugurlu) [inline BitToGithub/Program.cs]: I would remove commented out lines.

File: BitToGithub/Program.cs [hunk 2]
```diff
@@ -50,7 +54,9 @@ static void Main(string[] args)
                     int startIndex = 1;
                     while (true)
                     {
-                        var response = client.GetAsync(bbRepoIssuesUrl + "?start=\{startIndex}").Result;
+                        System.Threading.Thread.Sleep(3000);
+                        //var response = client.GetAsync(bbRepoIssuesUrl + "?start=\{startIndex}").Result;
```
Comment (tugberkugurlu) [inline BitToGithub/Program.cs]: I would remove commented out lines.

File: BitToGithub/Program.cs [hunk 3]
```diff
@@ -50,7 +54,9 @@ static void Main(string[] args)
                     int startIndex = 1;
                     while (true)
                     {
-                        var response = client.GetAsync(bbRepoIssuesUrl + "?start=\{startIndex}").Result;
+                        System.Threading.Thread.Sleep(3000);
```
Comment (tugberkugurlu) [inline BitToGithub/Program.cs]: Do we need this?

File: BitToGithub/Program.cs [hunk 4]
```diff
@@ -61,7 +67,12 @@ static void Main(string[] args)
                                 Console.WriteLine("found issue {0}", issue.local_id);
                                 var newIssue = new NewIssue(issue.title);
                                 newIssue.Labels.Add(migratedLabel);
-                                newIssue.Body = issue.content + "\{Environment.NewLine}\{Environment.NewLine}> Originally created at \{issue.utc_created_on} (UTC) by \{issue.reported_by.username} as a(n) \{issue.priority} issue.";
+                                //newIssue.Body = issue.content +  "\{Environment.NewLine}\{Environment.NewLine}> Originally created at \{issue.utc_created_on} (UTC) by \{issue.reported_by.username} as a(n) \{issue.priority} issue.";
```
Comment (tugberkugurlu) [inline BitToGithub/Program.cs]: I would remove commented out lines.

File: BitToGithub/Program.cs [hunk 5]
```diff
@@ -61,7 +67,12 @@ static void Main(string[] args)
                                 Console.WriteLine("found issue {0}", issue.local_id);
                                 var newIssue = new NewIssue(issue.title);
                                 newIssue.Labels.Add(migratedLabel);
-                                newIssue.Body = issue.content + "\{Environment.NewLine}\{Environment.NewLine}> Originally created at \{issue.utc_created_on} (UTC) by \{issue.reported_by.username} as a(n) \{issue.priority} issue.";
```
Comment (tugberkugurlu) [inline BitToGithub/Program.cs:64]: Nice, thanks for removing these. I believe these are pre-release way to have Interpolated Strings which has changed. 

You can use https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/interpolated-strings if you prefer to.

**Comment by tugberkugurlu** [comment]
Thanks for this 🎉 really appreciate it @OkumaScott! I suggested a few improvements, hope that they make sense 😄

File: BitToGithub/Program.cs (hunk 3 above)
Comment (ssolmer) [reply BitToGithub/Program.cs]: I suppose one sleep is enough... I used the script debugging in break mode to transfer my issues, so I didn't experience the GitHub blocking behavior. Was just erring on the safe side.

File: BitToGithub/Program.cs (hunk 5 above)
Comment (ssolmer) [reply BitToGithub/Program.cs:64]: I spend a lot of time being forced to code for CLR2 compatibility, so often miss out on all the fancy new sugar.

**Pull request closed without merging**